app/controllers/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    books = Book.all.page(1)
```

Comment:
This will always bring the first page, and won't be modifiable in the request, API should be able to handle any page request